src/st_widgets.c

```diff
@@ -142,12 +144,19 @@ static void UpdateSecretMessage(sbe_widget_t *widget, player_t *player)
         return;
     }
 
-    static char string[80];
```

Comment:
The level title won't be displayed if `hud_secret_message == 0`.

Reply:
Fixed in 85bc7ef

Reply:
> As it is now, the "secret revealed" message immediately overrides the map announcement if the player starts the map in a sector tagged as secret. Is this intended?

Yes, it's intended. For me, the secret message is more important than the level title, but we can change the priority if you prefer. By the way, what is this map?

>  Could we show both messages on two lines probably?

It won't look good.

Reply:
> By the way, what is this map?

A copy of MAP01 with the starting sector tagged as secret. :grin: 

I have seen a map like this in the wild, but couldn't remember which it was.

Reply:
BTW, why do we clear the line and add the string again during each tic for this widget? The strings aren't going to change anyway.

Reply:
Strings can change, e.g. if the player enters a secret sector while the map announce is still on. Note that no strings are copied every tic, only pointers.